user-guide/Advanced_Modules/EPM/Tutorials/EPM_Introduction_Tutorial.md

```diff
@@ -0,0 +1,155 @@
+---
+uid: EPM_Introduction_Tutorial
+---
+
+# Getting started with EPM
+
+This tutorial will provide a brief introduction on the EPM architecture and how you can use many EPM specific features.
+
+Expected duration: 30 minutes
+
+> [!NOTE]
+> The content and screenshots for this tutorial have been created using DataMiner version 10.4.5.
+
+## Prerequisites
+
+- A DataMiner System that is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
```

Comment:
As the [Topology app](https://docs.dataminer.services/develop/DMI/EPM_Integration_Training/Topology_app_configuration.html) is used, should the CPEIntegration soft-launch option also be enabled on the system? Or does this happen automatically when the package is deployed?

Reply:
Indeed, I forgot to mention that. Though the softlaunch option is not needed if working on a DaaS